src/server.ts

```diff
@@ -124,30 +125,50 @@ createConnection(dbOptions).then(async connection => {
     logger.error("shutting down notifier due to un-successful database connection...")
     process.exit(1)
 });
-const natsEventHandler = (msg: string) => {
+
+const natsEventHandler = async (msg: string) => {
     const eventAsString = JSON.parse(msg)
     const event = JSON.parse(eventAsString) as Event
-    notificationService.sendNotification(event)
+    const response = await notificationService.sendNotification(event)
+    if (response.status != 0){
+        successNotificationMetricsCounter.inc()
+    } else{
+        failedNotificationMetricsCounter.inc()
+    }
 }
-app.get('/', (req, res) => res.send('Welcome to notifier Notifier!'))
+
+app.get('/', (req, res) => {
+    httpRequestMetricsCounter.labels({method: req.method, endpoint: req.url, statusCode: res.statusCode}).inc()
+    res.send('Welcome to notifier Notifier!')
```

Comment:
```suggestion
    return res.send('Welcome to notifier Notifier!')
```

Reply:
res.send is handling it, not required to return